concept/thing/impl/AttributeImpl.java

```diff
@@ -88,9 +89,13 @@ public FunctionalIterator<ThingImpl> getOwners() {
 
     @Override
     public FunctionalIterator<ThingImpl> getOwners(ThingType ownerType) {
-        return ownerType.getSubtypes().map(ot -> ((ThingTypeImpl) ot).vertex).flatMap(
-                v -> readableVertex().ins().edge(HAS, (PrefixIID.of(v.encoding().instance())), v.iid()).from()
-        ).map(ThingImpl::of);
+        return ownerType.getSubtypes().flatMap(this::getOwnersExplicit);
+    }
```

Comment:
The insides of the `map`s are replicated within `getOwnersExplicit`, and we can use it here instead.